pennylane/qnode_new/jacobian.py

```diff
@@ -34,6 +34,8 @@ def __init__(self, func, device, mutable=True, properties=None):
         """dict[int, str]: map from flattened quantum function positional parameter index
         to the gradient method to be used with that parameter"""
 
+    metric_tensor = None
```

Comment:
because the TF and Torch interfaces still expect all QNodes to have this attribute 🤦‍♂ 
This can be removed once the TF and Torch interfaces have been updated for the QNode refactor.